src/commands/createFunctionApp/FunctionAppCreateStep.ts

```diff
@@ -80,11 +80,13 @@ export class FunctionAppCreateStep extends AzureWizardExecuteStep<IFunctionAppWi
             this.addCustomLocationProperties(site, context.customLocation);
         }
 
-        const isNotFree = context.plan?.sku?.family?.toLowerCase() !== 'u';
-        const isNotElasticPremium = context.plan?.sku?.tier?.toLowerCase() !== 'ep';
-        const isNotConsumption: boolean = context.plan?.sku?.family?.toLowerCase() !== 'y';
-        if (isNotFree && isNotElasticPremium && isNotConsumption) {
-            nonNullProp(site, 'siteConfig').alwaysOn = true;
+        if (context.plan?.sku?.family !== undefined) {
+            const isNotFree = context.plan?.sku?.family?.toLowerCase() !== 'u';
```

Comment:
Inside this `if` block you no longer need to use the `?.`, since `context.plan.sku.family` has already been determined to be defined. TypeScript also knows this so it won't complain.